### ISSUE TITLE
Extending eslint doesn't work

### ISSUE THREAD

**Issue by epicfaace** [google/gts#465]
I can't extend the eslint config for gts because it seems like the latest npm package doesn't have `.eslintrc.json` published.

I tried, as in the readme:

```
  "eslintConfig": {
    "extends": "./node_modules/gts"
  }
```

@JustinBeckwith ?

**Comment by thomasw** [comment]
I ran into this a few days ago myself. The eslint configuration described in the master version of the readme hasn't been released yet: https://github.com/google/gts/compare/v1.1.2...master

The last published release still uses tslint and does not include the eslint configs you're trying to extend: https://github.com/google/gts/tree/v1.1.2

**Comment by JustinBeckwith** [comment]
Correct.  We're furiously trying to get `2.0` out the door this week.  For now, you can `npm install gts@next` and it should pin you to the right dist tag until we're out :)

**Comment by thomasw** [comment]
Thanks, @JustinBeckwith !

**Comment by brokenthorn** [comment]
> Correct. We're furiously trying to get `2.0` out the door this week. For now, you can `npm install gts@next` and it should pin you to the right dist tag until we're out :)

Thank you!

After `2.0.0` is released, besides changing `package.json` from the alpha or beta version to version `^2.0.0` of `gts`, is there anything else we will have to? Like for example upgrade any other packages?

**Comment by JustinBeckwith** [comment]
Hey folks!  We just shipped 2.0 :)  You can go back to the `latest` tag now if you want to upgrade!  

On the question about other changes - I wrote an upgrade guide here:
https://github.com/google/gts/releases/tag/v2.0.0

There are many breaking changes 😬
